### PR TITLE
NAS-115243 / 22.02.1 / Dashboard Pool Widget not seeing new pool

### PR DESCRIPTION
**Pull request by AlexKarpov98**
Testing:

Create a new Pool or more
Go to Dashboard -> Click Configure
Please confirm if displays all the pools

### CODE REVIEW

File: src/app/pages/dashboard/components/dashboard/dashboard.component.ts [hunk 1]
```diff
@@ -619,11 +619,21 @@ export class DashboardComponent implements OnInit, AfterViewInit, OnDestroy {
...
+        return {
+          ...widget,
+          rendered: false,
+        };
+      }
+      return widget;
+    });
 
-    this.setDashState([...state, ...hidden]);
+    this.setDashState(dashState);
```
Comment (bvasilenko) [inline src/app/pages/dashboard/components/dashboard/dashboard.component.ts:636]: I noticed that we've lost ability to drag widgets around to reorder them. You can check this feature by clicking "Reorder" button on the dashboard.

**Review by bvasilenko — CHANGES_REQUESTED** [review]

File: src/app/pages/dashboard/components/dashboard/dashboard.component.ts (hunk 1 above)
Comment (AlexKarpov98) [reply src/app/pages/dashboard/components/dashboard/dashboard.component.ts]: Sorry, fixed filtering condition

**Comment by bvasilenko** [comment]
I confirm that Pool widget is able to display multiple pools:

image
image


+ the dashboard widgets can be reordered now

**Comment by Codelica** [comment]
Does this patch also cover missing network interface widgets?   If not, I'll file a separate bug in JIRA.  Thanks!

**Pull request merged**